package.json

```diff
@@ -48,6 +48,8 @@
     "create-react-class": "15.6.3",
     "d3": "3.5.16",
     "date-fns": "1.30.1",
+    "downshift": "3.2.10",
+    "react-draggable": "3.2.1",
```

Comment:
Why do we need these two?

Reply:
Oops, we don't. This is a leftover from when I was testing with a local build of ui-kit